src/sentry/tasks/files.py

```diff
@@ -88,3 +89,37 @@ def delete_unreferenced_blobs(blob_model, blob_index_model, blob_ids):
                 # Do nothing if the blob was deleted in another task, or
                 # if had another reference added concurrently.
                 pass
+
+
+@instrumented_task(
+    name="sentry.tasks.files.copy_to_control",
+    queue="files.copy",
+    default_retry_delay=60 * 5,
+    max_retries=MAX_RETRIES,
+    autoretry_for=(DatabaseError, IntegrityError),
+    acks_late=True,
+)
+def copy_file_to_control_and_update_model(
+    app_name: str, model_name: str, model_id: int, file_id: int
```

Comment:
It's recommended to also accept `**kwargs`, for better compatibility (https://develop.sentry.dev/services/queue/#registering-a-task)